tests/test_vms_exist.py

```diff
@@ -20,17 +25,17 @@ def tearDown(self):
         pass
 
     def test_expected(self):
-        vm_set = set(self.app.domains)
-        for test_vm in WANTED_VMS:
-            self.assertIn(test_vm, vm_set)
+        sdw_tagged_vm_names = [vm.name for vm in self.sdw_tagged_vms]
+        expected_vms = set(SD_VMS + SD_DVM_TEMPLATES + SD_TEMPLATES)
+        self.assertEqual(set(sdw_tagged_vm_names), set(expected_vms))
```

Comment:
Just noting for our future selves: Sometimes the set of sd-tagged vms isn't exactly equal to the set of VMs expressed here, but they're all edge cases and shouldn't interfere:
- devs may tag their sd-dev vm @tag:sd-workstation if they follow our (old?) developer guidelines. This shouldn't be a problem though because those environments aren't where we should be running `make test` since they're not indicative of real conditions
- prod instances may(?) configure additonal VMs and tag them with @sd-workstation to take advantage of our strict rpc policies. I don't think this is happening, but in any case, prod setups shouldn't be running dom0 tests.